crowdstrikeoauthapi_connector.py

```diff
@@ -1556,6 +1556,82 @@ def _handle_remove_hosts(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS, "Host removed successfully")
 
+    def _handle_create_ioa_rule_group(self, param):
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        create_params = {
+            'name': param['name'],
+            'description': param['description'],
+            'platform': param['platform']
+        }
+        ret_val, resp_json = self._make_rest_call_helper_oauth2(
+            action_result,
+            CROWDSTRIKE_IOA_CREATE_RULE_GROUP_ENDPOINT,
+            json_data=create_params,
+            method='post'
+        )
+
+        if phantom.is_fail(ret_val):
+            return action_result.get_status()
+
+        rulegroup_id = resp_json['resources'][0]['id']
+        rulegroup_version = resp_json['resources'][0]['version']
```

Comment:
This and line above, you can use .get() to be safe

Reply:
Good catch @alexa-phantom I missed this

Reply:
Yeah good call, though if either value turns out to be `None`, we'll have to throw an error anyways (because those values are required for the later update call).

I'll revise that tomorrow.

Reply:
Added a check that will raise an error if either of these values are missing. Waiting for tests to re-run.

Not sure how to get test coverage here, since I'm not aware of any circumstance where this will happen: either CrowdStrike creates the rule group, answers 2xx, and returns the id/version.... or it doesn't create, and answers 4xx/5xx.